electrumx/server/controller.py

```diff
@@ -102,6 +102,8 @@ def __init__(self, env):
         self.mempool = MemPool(self.bp, self)
         self.peer_mgr = PeerManager(env, self)
 
+        self.tcp_server_started = asyncio.Event()
```

Comment:
Can we perhaps call this member "listening" instead?  I don't want to name it in a way it might cause expectation of an SSL event too.

You should add a comment about its purpose.

Also please submit the event and the Env changes as separate PRs.